### ISSUE TITLE
False positive for apostrophe in words

### ISSUE THREAD

**Issue by maximousblk** [sindresorhus/awesome-lint#104]
for words like `don't` or `isn't` I get the following error:

```
×   941:135  "’" is used without matching "‘"            remark-lint:match-punctuation
```

**Comment by maximousblk** [comment]
I think you can solve this by looking for blankspace after the apostrophe.

This would create a new issue that it will throw an error if the text is similar to ` kids' book `. But this is also a very less used grammar.

**Comment by kdeldycke** [comment]
You should use the ASCII straight apostrophe, i.e. `'` (char code 39, HEX 27) instead of `’ `, which is classified by linter's rules as a quote and need to be matched.

See: https://en.wikipedia.org/wiki/Apostrophe

**Comment by kdeldycke** [comment]
I guess this issue can be closed now: all rules related to quotes are implemented by rules.

**Comment by jtojnar** [comment]
@kdeldycke U+2019 is the preferred character for punctuation apostrophe, see https://en.wikipedia.org/wiki/Apostrophe#Unicode so the linter rules are wrong and should either be adjusted or disabled. We are not using typewriters any more.
